src/Faker/Provider/Image.php

```diff
@@ -44,9 +52,19 @@ public static function imageUrl(
         $category = null,
         $randomize = true,
         $word = null,
-        $gray = false
+        $gray = false,
+        $format = 'png'
     ) {
-        $size = sprintf('%dx%d.png', $width, $height);
+        // Validate image format
+        if (!in_array(strtolower($format), static::$imageFormats)) {
+            throw new \InvalidArgumentException(sprintf(
+                'Invalid image format "%s". Allowable formats are: %s',
+                $format,
+                implode(', ', static::$imageFormats)
```

Comment:
Implement the constants as reference here

Reply:
I implemented the constants in getter functions so I could access them from the unit tests. Let me know if this is acceptable.